pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -416,19 +343,19 @@ func (r *MigPlan) UpdateRegistryDC(storage *MigStorage, deploymentconfig *appsv1
 		}
 		envFrom = append(envFrom, source)
 	}
-	deploymentconfig.Spec = appsv1.DeploymentConfigSpec{
-		Replicas: 1,
-		Selector: map[string]string{
-			"app":              name,
-			"deploymentconfig": name,
-		},
-		Strategy: appsv1.DeploymentStrategy{Resources: kapi.ResourceRequirements{}},
-		Template: &kapi.PodTemplateSpec{
+
+	deployment.Spec = appsv1.DeploymentSpec{
+		Replicas: pointer.Int32Ptr(1),
+		Selector: metav1.SetAsLabelSelector(map[string]string{
+			"app":        name,
+			"deployment": name,
+		}),
+		Template: kapi.PodTemplateSpec{
 			ObjectMeta: metav1.ObjectMeta{
 				CreationTimestamp: metav1.Time{},
 				Labels: map[string]string{
-					"app":              name,
-					"deploymentconfig": name,
+					"app":        name,
+					"deployment": name,
```

Comment:
add this here as well https://github.com/konveyor/mig-controller/pull/681/commits/782c73e99b23ced4d99347c98ee541a9cfc922c1#r497020064

Reply:
Done